procfs-core/src/netstat.rs

```diff
@@ -0,0 +1,625 @@
+            tcp_ext_busy_poll_rx_packets: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_auto_corking: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_from_zero_window_adv: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_to_zero_window_adv: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_want_zero_window_adv: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_syn_retrans: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_orig_data_sent: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_hystart_train_detect: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_hystart_train_cwnd: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_hystart_delay_detect: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_hystart_delay_cwnd: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_ack_skipped_syn_recv: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_ack_skipped_paws: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_ack_skipped_seq: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_ack_skipped_finwait2: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_ack_skipped_timewait: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_ack_skipped_challenge: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_win_probe: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_keep_alive: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_mtup_fail: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_mtup_success: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_delivered: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_delivered_ce: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_ack_compressed: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_zero_window_drop: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_rcv_qdrop: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_wqueue_too_big: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_fast_open_passive_alt_key: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_timeout_rehash: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_duplicate_data_rehash: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_ds_ack_recv_segs: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_ds_ack_ignored_dubious: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_migrate_req_success: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_migrate_req_failure: from_str!(u64, expect!(tcp_ext.next())),
+            tcp_ext_tcp_ecn_rehash: from_str!(u64, expect!(tcp_ext.next())),
```

Comment:
My 6.1.0 kernel doesn't have this field:

```
---- net::tests::test_netstats stdout ----
thread 'net::tests::test_netstats' panicked at procfs/src/net.rs:264:35:
called `Result::unwrap()` on an `Err` value: InternalError(bug at procfs-core/src/netstat.rs:534 (please report this procfs bug)
Internal Unwrap Error: NoneError)
```

We're going to want to find some older kernels to test this on.  I'll try to capture some examples of `/proc/net/netstat` on some random old machines

Reply:
Thank you! I'll provide the info about the kernel version I took it from

Reply:
@eminence 
I've confirmed that these parameters 
```
TcpExt:....
TcpDuplicateDataRehash TCPDSACKRecvSegs TCPDSACKIgnoredDubious TCPMigrateReqSuccess TCPMigrateReqFailure TCPECNRehash
```
come from 
```
sh-4.2$ uname -r
5.10.205-195.807.amzn2.x86_64
```
Not sure if that's too representative/widely used